packages/z2s/src/compiler-bigint.pg-test.ts

```diff
@@ -254,7 +254,7 @@ describe('compiling ZQL to SQL', () => {
     test('bigint exceeds safe range', async () => {
       const query = issueQuery.related('comments');
       const c = compile(ast(query), schema.tables);
-      const sqlQuery = formatPg(c);
+      const sqlQuery = formatPgInternalConvert(c);
```

Comment:
Would be good to add a test case for a where condition involving a bigint column.